### ISSUE TITLE
Leads do not record unless post-name is set as permalink.

### ISSUE THREAD

**Issue by atwellpub** [inboundnow/landing-pages#159]
I would like to investigate this and create a workflow that will store leads even when non post-name permalinks are being used. 

Solving this issue will prevent  future customer support requests.

More details here:
http://support.inboundnow.com/support/topic/some-leads-show-in-lead-management-most-dont/#post-114253

**Comment by DavidWells** [comment]
Good call. I just looked around class.lead-storage and can't put finger on where the error is happening.

Like there isn't a return statement anywhere based on the permalink/post type.

Do we return false anywhere based on url/post type? Or maybe a wordpress error is being thrown. 

I think the return data from the ajax call might be a good place to look. It should return numerical post ID

**Comment by atwellpub** [comment]
Kirit checked into this and reported no bug. Lead stores when permalinks are set to default. I also checked behind Kirit and confirmed. Which had me wondering what the known issue was and I remembered the issue we had with permalinks is that rotation was impossible if permalinks were set to default. 

I've added this alert in to notify users if their permalink settings are set to default. It is a hard, inclosable alert as I currently can't imagine any reason to have default permalinks in today's environments. 

https://github.com/inboundnow/landing-pages/commit/6c8d82943dd61e75e3dc3a68ce999963aa0e70f2
